PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -4,70 +4,79 @@ This document provides information on how to get your repository set up to take
 
-### Metadata ingest and crosswalk
+3. If there are fulltext files associated, they will be attached to the item.
 
-All metadata will be converted to EPrints metadata following the rules laid down in the Atom XSLT import plugin.
-This crosswalk is part of your EPrints souce, and can be found in:
+4. The item will be left in the sword user's workarea or review queue, depending on the deposit location that has been selected in PubRouter Admin page.
 
-    /opt/eprints3/perl_lib/EPrints/Plugin/Import/XSLT/Atom.xsl
+## DSpace
 
-By default it will crosswalk the absolute basic Atom metadata, so you may wish to modify this file to fit with your EPrints schema as you see fit.
+Router is certified to work with Dspace 5.x
 
-In order that you do not lose any information if your XSLT does not yet do what you want, Router will also deposit
-a full copy of the Atom Entry XML as a file attached to the eprint.  For information about the metadata held in that
-file see the [Crosswalk Documentation](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/sword-out/XWALK.md#jper-core-metadata-to-dublin-corerioxx-xml)
+Dspace needs to be configured and enabled following these instructions: [SWORDv2 Server](https://wiki.duraspace.org/display/DSDOC5x/SWORDv2+Server)
 
-## DSpace/Other
 
-Router has so far had limited use on DSpace, so documentation is subject to change.
+### Deposit process
 
-### How the deposit will look
+The behaviour during individual notification deposit is as follows:
 
-The behaviour during individual notification deposit you will see is as follows:
+1. Metadata will be deposited as a new item into the collection you specified in your account settings.
 
-1. Metadata will be deposited and crosswalked as per the DSpace SWORDv2 configuration (see below) into the collection you specified in your account
+2. The metadata may also be supplied as an XML file attached to the item (if provided by publisher)
 
-2. If there are fulltext files associated, they will be attached to the Item
+3. If there are fulltext files associated, they will be attached to the item.
 
-3. Upon completion, the item will be injected into the collection's workflow
+4. Upon completion, the item will be injected into the collection's workflow
  
-### Metadata ingest and crosswalk
+### Metadata customisation
 
 All the metadata will be converted to DSpace Item metadata following the rules laid down in the swordv2-server.cfg file, which
 can be found in
 
     dspace/config/modules/swordv2-server.cfg
     
-By default it will crosswalk a broad selection of Dublin Core fields, and you can customise that here.  If you wish to also support RIOXX you may need to
-extend your EntryIngester implementation.
+By default it will crosswalk a broad selection of Dublin Core fields, but this may be customised. See [here](https://wiki.duraspace.org/display/DSDOC5x/Metadata+and+Bitstream+Format+Registries) for more information.
```

Comment:
By default it will crosswalk a broad selection... 
what is 'it' in this context? This isn't clear to me